cpp_modules/src/mcts/ValueHeuristicEnum.h

```diff
@@ -7,17 +7,25 @@
 
 enum class ValueHeuristic {
 	UCT,
-	Ucb1Normal2
+	UCB1Normal2,
```

Comment:
One thing worth taking into account is that we'll probably need to write in our paper about why we chose the different value heuristics. Maybe we should look into the ones who got the most reward from the "Scale-Adaptive balancing..." paper, and try the top 3-5 of those. That way, we have a source to refer to.

Reply:
I agree